### PR TITLE
DSD-853: Allow CardActions to align to the right in the Card component

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Fixes JIRA ticket [DSD-853](https://jira.nypl.org/browse/DSD-853)

## This PR does the following:

- Adds the `isAlignedRightActions` prop to the `Card` component so that all of its `CarActions` children move to the right of the main content area.
- This is set only in the row layout.

## How has this been tested?

Storybook

## Accessibility concerns or updates

- N/A

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] View [the example in Storybook]()

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr946-jnwdpwqqf0ojmloisto12wrprf0hua4s.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6255a9b6db46ec1959cb1b78

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr946-jnwdpwqqf0ojmloisto12wrprf0hua4s.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6255a9b6db46ec1959cb1b78

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr946-jnwdpwqqf0ojmloisto12wrprf0hua4s.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6255a9b6db46ec1959cb1b78

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr946-jnwdpwqqf0ojmloisto12wrprf0hua4s.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6255a9b6db46ec1959cb1b78

File: src/components/Card/Card.tsx [hunk 1]
```diff
@@ -313,6 +325,14 @@ export const Card = chakra((props: React.PropsWithChildren<CardProps>) => {
         <Box className="card-body" __css={styles.body}>
           {cardContents}
         </Box>
+        {cardRightContents.length ? (
+          <Box
+            className="card-right"
+            __css={{ ...styles.body, marginLeft: "m" }}
```
Comment (bigfishdesign13) [inline src/components/Card/Card.tsx]: The styles for this container need to be cleaned up.

Mobile View
- `width` should be `100%`
- `marginLeft` should be  `0`
- `marginTop` should be  `xs` (8px)

Desktop View
- `width` should be `180px`
- `marginLeft` should be `m`
- `marginTop` should be  `0`

**Review by bigfishdesign13 — CHANGES_REQUESTED** [review]
Just need to clean up the styles for the new “action on right side” container.

**Comment by EdwinGuzman** [comment]
Updated. FYI on how it looks with multiple buttons or links but I think it would be the responsibility of the content that is added rather than the `CardActions` component:
Screen Shot 2022-04-15 at 4 03 30 PM
Screen Shot 2022-04-15 at 4 03 17 PM

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr946-jnwdpwqqf0ojmloisto12wrprf0hua4s.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6255a9b6db46ec1959cb1b78

**Review by bigfishdesign13 — APPROVED** [review]
Looks good.  And yes, I agree.  The consuming app should be responsible for formatting their content to fit properly in a small space.  Of course, I think providing proper layout components to do that is our responsibility.  And we are getting there.

**Pull request merged**